tests/unit/model/user.model.spec.js

```diff
@@ -27,14 +27,18 @@ describe('User model', () => {
       const server = '/cylc/user/john.foe'
       const owner = 'john.goe'
       const permissions = ['play', 'ping', 'read']
-      const user = new User(name, groups, created, admin, server, owner, permissions)
+      const mode = 'single user'
+      const initials = 'jf'
+      const user = new User(name, groups, created, admin, server, owner, permissions, mode, initials)
       expect(user.username).to.equal(name)
       expect(user.groups.length === 2).to.equal(admin)
       expect(user.admin).to.equal(true)
       expect(user.created).to.equal(created)
       expect(user.server).to.equal(server)
       expect(user.owner).to.equal(owner)
       expect(user.permissions).to.equal(permissions)
+      expect(user.mode).to.equal(mode)
+      expect(user.initials).to.equal(initials)
     })
```

Comment:
There are more things coming back from the userprofile endpoint now although they are not actually being used, for example...
avatar_url
color
should these be added to the Vue user model and service and tests as well?

Reply:
Maybe, we can do that later.

Opened an issue for it: https://github.com/cylc/cylc-ui/issues/1519